CRITICAL - 2024-10-10 02:56:43 --> ErrorException: Undefined variable $errors
[Method: GET, Route: /]
in APPPATH/Views/auth/login.php on line 41.
 1 APPPATH/Views/auth/login.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $errors', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/login.php', 41)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/login.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('auth/login', [], true)
 5 APPPATH/Controllers/AuthController.php(131): view('auth/login')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->loginView()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 02:56:46 --> ErrorException: Undefined variable $errors
[Method: GET, Route: /]
in APPPATH/Views/auth/login.php on line 41.
 1 APPPATH/Views/auth/login.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $errors', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/login.php', 41)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/login.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('auth/login', [], true)
 5 APPPATH/Controllers/AuthController.php(131): view('auth/login')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->loginView()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 02:56:58 --> ErrorException: Undefined variable $errors
[Method: GET, Route: /]
in APPPATH/Views/auth/login.php on line 41.
 1 APPPATH/Views/auth/login.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $errors', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/login.php', 41)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/login.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('auth/login', [], true)
 5 APPPATH/Controllers/AuthController.php(131): view('auth/login')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->loginView()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 03:00:49 --> ErrorException: Attempt to read property "email" on array
[Method: POST, Route: auth/login]
in APPPATH/Controllers/AuthController.php on line 57.
 1 APPPATH/Controllers/AuthController.php(57): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "email" on array', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php', 57)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->login()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 03:57:08 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(27): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:00:46 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(28): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:02:47 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(28): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:03:27 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(28): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:04:56 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(27): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:05:06 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(27): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:06:42 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(26): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:10:20 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(26): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:10:22 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(26): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:10:23 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(26): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:10:24 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(26): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:10:24 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(26): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:11:32 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(27): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:12:24 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(27): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:12:26 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(27): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 04:12:40 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/AuthController.php(27): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\AuthController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
CRITICAL - 2024-10-10 04:17:59 --> ErrorException: Undefined variable $filters
[Method: GET, Route: auth/display]
in APPPATH/Views/auth/display.php on line 9.
 1 APPPATH/Views/auth/display.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $filters', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/display.php', 9)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('auth/display', [], true)
 5 APPPATH/Controllers/AuthController.php(72): view('auth/display')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->users()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 04:18:29 --> ErrorException: Undefined variable $filters
[Method: GET, Route: auth/display]
in APPPATH/Views/auth/display.php on line 9.
 1 APPPATH/Views/auth/display.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $filters', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/display.php', 9)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('auth/display', [], true)
 5 APPPATH/Controllers/AuthController.php(72): view('auth/display')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->users()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 04:19:00 --> ErrorException: Undefined variable $filters
[Method: GET, Route: auth/display]
in APPPATH/Views/auth/display.php on line 9.
 1 APPPATH/Views/auth/display.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $filters', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/display.php', 9)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/auth/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('auth/display', [], true)
 5 APPPATH/Controllers/AuthController.php(72): view('auth/display')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->users()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 05:50:57 --> ErrorException: Undefined array key "nombres"
[Method: GET, Route: empleado/display]
in APPPATH/Views/empleado/display.php on line 27.
 1 APPPATH/Views/empleado/display.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombres"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/empleado/display.php', 27)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/empleado/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('empleado/display', [], true)
 5 APPPATH/Controllers/EmpleadoController.php(45): view('empleado/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\EmpleadoController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 05:50:59 --> ErrorException: Undefined array key "nombres"
[Method: GET, Route: empleado/display]
in APPPATH/Views/empleado/display.php on line 27.
 1 APPPATH/Views/empleado/display.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombres"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/empleado/display.php', 27)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/empleado/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('empleado/display', [], true)
 5 APPPATH/Controllers/EmpleadoController.php(45): view('empleado/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\EmpleadoController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 06:13:30 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: empleado/findone/1]
in APPPATH/Views/empleado/profile.php on line 82.
 1 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('empleado/profile', [], true)
 3 APPPATH/Controllers/EmpleadoController.php(93): view('empleado/profile', [...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\EmpleadoController->findone('1')
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 18:24:50 --> ErrorException: Undefined array key "nombres"
[Method: GET, Route: departamento/display]
in APPPATH/Views/departamento/display.php on line 26.
 1 APPPATH/Views/departamento/display.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombres"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/departamento/display.php', 26)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/departamento/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('departamento/display', [], true)
 5 APPPATH/Controllers/DepartamentoController.php(15): view('departamento/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\DepartamentoController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DepartamentoController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 18:28:42 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `de...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `de...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `de...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `de...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/DepartamentoController.php(29): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\DepartamentoController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DepartamentoController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 18:30:27 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `de...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `de...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `de...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `de...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/DepartamentoController.php(29): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\DepartamentoController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DepartamentoController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
ERROR - 2024-10-10 18:58:02 --> mysqli_sql_exception: Unknown column 'empleados.nombres' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `permiso...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `permiso...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `permiso...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `permiso...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/PermisosModel.php(23): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/PermisoController.php(15): App\Models\PermisosModel->getPermisosWithEmpleados()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\PermisoController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PermisoController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 18:58:02 --> Error: Call to a member function getResult() on false
[Method: GET, Route: permiso/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/PermisosModel.php(23): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/PermisoController.php(15): App\Models\PermisosModel->getPermisosWithEmpleados()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\PermisoController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PermisoController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 20:29:31 --> BadMethodCallException: Call to undefined method App\Models\HorasExtrasModel::gethoraextrasWithEmpleados
[Method: GET, Route: horaextra/display]
in SYSTEMPATH/Model.php on line 932.
 1 APPPATH/Controllers/HorasExtrasController.php(15): CodeIgniter\Model->__call('gethoraextrasWithEmpleados', [])
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 20:31:30 --> mysqli_sql_exception: Unknown column 'horas_extras.detalle' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `horas_e...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `horas_e...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `horas_e...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `horas_e...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/HorasExtrasModel.php(23): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/HorasExtrasController.php(15): App\Models\HorasExtrasModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 20:31:30 --> Error: Call to a member function getResult() on false
[Method: GET, Route: horaextra/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/HorasExtrasModel.php(23): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/HorasExtrasController.php(15): App\Models\HorasExtrasModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 20:31:31 --> mysqli_sql_exception: Unknown column 'horas_extras.detalle' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `horas_e...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `horas_e...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `horas_e...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `horas_e...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/HorasExtrasModel.php(23): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/HorasExtrasController.php(15): App\Models\HorasExtrasModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 20:31:31 --> Error: Call to a member function getResult() on false
[Method: GET, Route: horaextra/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/HorasExtrasModel.php(23): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/HorasExtrasController.php(15): App\Models\HorasExtrasModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 20:31:47 --> mysqli_sql_exception: Unknown column 'horas_extras.detalle' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `horas_e...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `horas_e...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `horas_e...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `horas_e...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/HorasExtrasModel.php(23): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/HorasExtrasController.php(15): App\Models\HorasExtrasModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 20:31:47 --> Error: Call to a member function getResult() on false
[Method: GET, Route: horaextra/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/HorasExtrasModel.php(23): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/HorasExtrasController.php(15): App\Models\HorasExtrasModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 20:49:05 --> ErrorException: Undefined array key "detalle"
[Method: GET, Route: horaextra/display]
in APPPATH/Views/horaextra/display.php on line 29.
 1 APPPATH/Views/horaextra/display.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "detalle"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/horaextra/display.php', 29)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/horaextra/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('horaextra/display', [], true)
 5 APPPATH/Controllers/HorasExtrasController.php(16): view('horaextra/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\HorasExtrasController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorasExtrasController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 20:58:44 --> BadMethodCallException: Call to undefined method App\Models\PrestamoModel::getPermisosWithEmpleados
[Method: GET, Route: prestamo/display]
in SYSTEMPATH/Model.php on line 932.
 1 APPPATH/Controllers/PrestamoController.php(15): CodeIgniter\Model->__call('getPermisosWithEmpleados', [])
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\PrestamoController->display()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PrestamoController))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:20:56 --> BadMethodCallException: Call to undefined method App\Models\DescuentosPagosModel::getAll
[Method: GET, Route: cobro/display]
in SYSTEMPATH/Model.php on line 932.
 1 APPPATH/Controllers/CobroController.php(15): CodeIgniter\Model->__call('getAll', [])
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:22:08 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:22:47 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:23:06 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:23:07 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:23:45 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:23:46 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:24:08 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:24:47 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:24:54 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:24:56 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:31:32 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:32:13 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:32:14 --> ErrorException: Undefined array key "created_at"
[Method: GET, Route: cobro/display]
in APPPATH/Views/cobro/display.php on line 31.
 1 APPPATH/Views/cobro/display.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "created_at"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php', 31)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/cobro/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('cobro/display', [], true)
 5 APPPATH/Controllers/CobroController.php(16): view('cobro/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CobroController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CobroController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
CRITICAL - 2024-10-10 21:56:38 --> ErrorException: Undefined array key "estadocivil"
[Method: GET, Route: nomina/display]
in APPPATH/Views/nomina/display.php on line 27.
 1 APPPATH/Views/nomina/display.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "estadocivil"', '/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/nomina/display.php', 27)
 2 SYSTEMPATH/View/View.php(234): include('/Applications/XAMPP/xamppfiles/htdocs/nomina/app/Views/nomina/display.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('nomina/display', [], true)
 5 APPPATH/Controllers/NominaController.php(16): view('nomina/display', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 21:58:15 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 21:58:15 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 21:59:20 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 21:59:20 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:01:12 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\nominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:01:12 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\nominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:01:39 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:01:39 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:02:05 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:02:05 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:02:05 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:02:05 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:02:09 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:02:09 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:03:19 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:03:19 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:03:27 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:03:27 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:06:54 --> mysqli_sql_exception: Unknown column 'nomina.descuento' in 'field list' in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `nomina`...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nomina`...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nomina`...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `nomina`...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\NominaController->display()
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}
CRITICAL - 2024-10-10 22:06:54 --> Error: Call to a member function getResult() on false
[Method: GET, Route: nomina/display]
in SYSTEMPATH/Model.php on line 287.
 1 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH/Models/NominaModel.php(22): CodeIgniter\BaseModel->findAll()
 3 APPPATH/Controllers/NominaController.php(15): App\Models\NominaModel->getAll()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\NominaController->display()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NominaController))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php')
ERROR - 2024-10-10 22:31:33 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`nomina`.`empleado`, CONSTRAINT `departamento_empleado` FOREIGN KEY (`departamento`) REFERENCES `departamento` (`id`)) in /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 /Applications/XAMPP/xamppfiles/htdocs/nomina/app/Controllers/EmpleadoController.php(23): CodeIgniter\BaseModel->save(Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(933): App\Controllers\EmpleadoController->register()
#10 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
#11 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /Applications/XAMPP/xamppfiles/htdocs/nomina/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 /Applications/XAMPP/xamppfiles/htdocs/nomina/system/rewrite.php(44): require_once('/Applications/X...')
#16 {main}
CRITICAL - 2024-10-10 22:59:04 --> ErrorException: Cannot use isset() on the result of an expression (you can use "null !== expression" instead)
[Method: GET, Route: nomina/display]
in APPPATH/Views/layout/header.php on line 36.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
